src/components/menu/notebook-header.jsx

```diff
@@ -11,6 +11,10 @@ import AppMessages from '../app-messages/app-messages'
 import IodideModalsRoot from '../modals/iodide-modals-root'
 
 const theme = createMuiTheme({
+  typography: {
+    useNextVariants: true,
+    suppressDeprecationWarnings: true,
```

Comment:
i must preface this by saying that i dont actually know how this code works, but in general, "solving" a deprecation warning problem by adding a flag called "suppressDeprecationWarnings" seems *very* unlikely to be an actual solution.

i'll let @hamilton look more deeply into this, but i'm inclined to reject this on that basis -- we want to actually update our code to follow the new bet practice for MUI so that we don't get mysterious breakage during some update down the road -- we don't just want to hide the problem